lib/matplotlib/backends/backend_agg.py

```diff
@@ -460,9 +460,14 @@ def draw(self):
         # acquire a lock on the shared font cache
         RendererAgg.lock.acquire()
 
+        toolbar = self.toolbar
         try:
+            if toolbar:
+                toolbar.set_cursor(cursors.WAIT)
             self.figure.draw(self.renderer)
         finally:
+            if toolbar:
+                toolbar.set_cursor(cursors.POINTER)
```

Comment:
Good catch, but AFAICT there's no `get_cursor` method available...
Actually from the point of view of Qt the best solution would probably be to use http://doc.qt.io/qt-5/qguiapplication.html#setOverrideCursor (because the entire GUI is blocked during the redraw (... actually there may be some argument for doing the drawing asynchronously I guess)) but I don't know whether other toolkits support something similar.